src/panels/lovelace/cards/hui-logbook-card.ts

```diff
@@ -114,53 +117,14 @@ export class HuiLogbookCard extends LitElement implements LovelaceCard {
   }
 
   protected firstUpdated(): void {
-    this._fetchPersonNames();
+    if (this.hass) {
+      this._fetchUserDone = this._fetchUserNames(this.hass);
+    }
   }
 
   protected updated(changedProperties: PropertyValues) {
     super.updated(changedProperties);
-    if (!this._config || !this.hass) {
-      return;
-    }
-
-    const configChanged = changedProperties.has("_config");
-    const hassChanged = changedProperties.has("hass");
-    const oldHass = changedProperties.get("hass") as HomeAssistant | undefined;
-    const oldConfig = changedProperties.get("_config") as LogbookCardConfig;
-
-    if (
-      (hassChanged && oldHass?.themes !== this.hass.themes) ||
-      (configChanged && oldConfig?.theme !== this._config.theme)
-    ) {
-      applyThemesOnElement(this, this.hass.themes, this._config.theme);
-    }
-
-    if (
-      configChanged &&
-      (oldConfig?.entities !== this._config.entities ||
-        oldConfig?.hours_to_show !== this._config!.hours_to_show)
-    ) {
-      this._logbookEntries = undefined;
-      this._lastLogbookDate = undefined;
-
-      if (!this._configEntities) {
-        return;
-      }
-
-      this._throttleGetLogbookEntries();
-      return;
-    }
-
-    if (
-      oldHass &&
-      this._configEntities!.some(
-        (entity) =>
-          oldHass.states[entity.entity] !== this.hass!.states[entity.entity]
-      )
-    ) {
-      // wait for commit of data (we only account for the default setting of 1 sec)
-      setTimeout(this._throttleGetLogbookEntries, 1000);
-    }
```

Comment:
these are moved to method `_updateData` because these needs to be run async after `_fetchUserDone` promise is resolved.